src/riscv.h

```diff
@@ -97,6 +98,8 @@ enum {
 
 #define BLOCK_MAP_CAPACITY_BITS 10
 
+#define PRIV(x) ((vm_attr_t *) x->userdata)
```

Comment:
The macro `PRIV` should not appear in public header, `riscv.h`.

Reply:
stored it in `riscv_private.h`